### ISSUE TITLE
WCAG 2.1解説書の公開準備

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#362]
- [x] Googleフォームへの問い合わせリンクを確認する（https://github.com/waic/wcag21/issues/235 ）
- [x] WCAG 2.1にjsで解説書へのリンクを挿入するようにする（https://github.com/waic/wcag21/issues/8 ）
- [x] GitHubでリリースタグをつけて、ファイルを固める
- [x] 公開するURLパスを決める
- [x] ステージング環境にアップロードを依頼する
- [x] WAICメーリングリストでWAIC内レビューを依頼する（レビュー期間2週間）
- [x] レビューコメントがあれば対応する
- [x] 本番環境で公開を依頼する
- [x] WAICサイトに掲載するニュースの文面を確定させる

**Comment by momdo** [comment]
https://github.com/waic/wcag21/pull/364 にて解説書へのリンクjs実施済み

**Comment by momdo** [comment]
#363 にて問い合わせリンクを設置済み。

**Comment by momdo** [comment]
公開パスは `/docs/WCAG21/Understanding/` の予定。

**Comment by momdo** [comment]
https://github.com/waic/wcag21/releases/tag/Understanding-202002 にてリリース済み。

**Comment by momdo** [comment]
WAIC内Slackにてアップロード依頼中。

**Comment by momdo** [comment]
-  `https://w3c.github.io/wcag/guidelines/` へのリンクがある（これは置換忘れ）
- `https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance` へのリンクがある（これはオリジナルファイルの誤り）

があるので要調整。

**Comment by momdo** [comment]
WCAG 2.1本体は一見リンクを付け足しただけ、に見えるが、内容の修正も入っているので差分の提示が必要か。

**Comment by momdo** [comment]
WCAG 2.1へのリンクは #367 にて対応済み。

**Comment by momdo** [comment]
https://github.com/waic/wcag21/releases/tag/wcag21-202002.1 WAIC内レビューにかける前に調整したものをリリース

**Comment by momdo** [comment]
WAIC内レビュー依頼済み。3/12（木）〆

**Comment by momdo** [comment]
3/12（木）17:00時点で、レビューコメントなし。

**Comment by momdo** [comment]
Google Doc（URLはWAICのSlack参照）に文面を作成中。

**Comment by SawadaStdDesign** [comment]
（レビューする時間を作れませんでしたので、公開後に見させていただきますね！）

**Comment by momdo** [comment]
3/13 9:00時点でのレビューコメントなし。

**Comment by momdo** [comment]
https://waic.jp/news/20200313/ 🎉
